Project doesn't compile (Rust 1.70)

Running `cargo build` from current main branch fails with error:

```
cargo build
   Compiling cargo-swift v0.5.1 (/Users/mo/code/cargo-swift)
error[E0659]: `console` is ambiguous
  --> src/lib.rs:35:9
   |
35 | pub use console::error::Result;
   |         ^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `console` could refer to a crate passed with `--extern`
   = help: use `::console` to refer to this crate unambiguously
note: `console` could also refer to the module defined here
  --> src/lib.rs:7:1
   |
7  | / pub(crate) mod console {
8  | |     mod command;
9  | |     pub mod config;
10 | |     pub mod error;
...  |
22 | |     pub use theme::*;
23 | | }
   | |_^
   = help: use `crate::console` to refer to this module unambiguously

error[E0659]: `console` is ambiguous
  --> src/lib.rs:36:9
   |
36 | pub use console::Config;
   |         ^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `console` could refer to a crate passed with `--extern`
   = help: use `::console` to refer to this crate unambiguously
note: `console` could also refer to the module defined here
  --> src/lib.rs:7:1
   |
7  | / pub(crate) mod console {
8  | |     mod command;
9  | |     pub mod config;
10 | |     pub mod error;
...  |
22 | |     pub use theme::*;
23 | | }
   | |_^
   = help: use `crate::console` to refer to this module unambiguously

For more information about this error, try `rustc --explain E0659`.
error: could not compile `cargo-swift` (lib) due to 2 previous errors
```

```shell
$ cargo version
cargo 1.70.0 (ec8a8a0ca 2023-04-25)
$ rustc --version
rustc 1.70.0 (90c541806 2023-05-31)
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -32,8 +32,8 @@ mod templating;
 mod xcframework;
 
 pub use commands::*;
-pub use console::error::Result;
-pub use console::Config;
+pub use crate::console::error::Result;
+pub use crate::console::Config;
 pub use lib_type::LibType;
 pub use targets::*;
 

